src/main/kotlin/nevam/User.kt

```diff
@@ -4,12 +4,18 @@ import nevam.util.GradleProperties
 
 data class NexusUser(val username: String, val password: String) {
   companion object {
-    fun readFrom(fileName: String): NexusUser {
+    fun readFrom(
+        fileName: String,
+        usernameProperty: String = DEFAULT_USERNAME_PROPERTY, passwordProperty: String = DEFAULT_PASSWORD_PROPERTY
```

Comment:
I'm under the assumption that binary compatibility doesn't matter for this project. Let me know if it does and if you'd prefer a `@JvmOverloads` or a manual overload to maintain the old function.

Reply:
Agree yeah that shouldn't be needed. If you get a chance to make changes, can you move the arguments to separate lines?